gemrb/core/Map.cpp

```diff
@@ -1395,8 +1395,18 @@ void Map::Shout(Actor* actor, int shoutID, unsigned int radius)
 			continue;
 		}
 
-		if (radius) {
-			if (Distance(actor->Pos, listener->Pos)>radius) {
+		if (!global) {
+			/* Audible range was confirmed to be 3x visual range in EEs, accounting for isometric scaling 
+			and more than 1x visual range in others in EE, the '48' (3*16) default can be set by the 'Audible Range'
+			game option in baldur.lua
+
+			This is a bit tricky, it has been show to not be very consistent. The game used a double value of visual 
+			range in several places, so we will use '3 * visual_range / 2' */
+			const Point& A = actor->Pos;
+			const Point& B = listener->Pos;
+			double angle = atan2(B.y - A.y, B.x - A.x);
+			int distance = (3 * actor->GetStat(IE_VISUALRANGE)) / 2;
+			if (Distance(A, B)>Feet2Pixels(distance, angle)) {
```

Comment:
```suggestion
			if (Distance(A, B) > Feet2Pixels(distance, angle)) {
```

Reply:
I completely agree with that suggestion. That is how I would have written it but I was emulating what was there before. As far as I understand it this project has loosely governed coding standards and primarily runs on mirroring the style of already implemented code. Would you say its better to just keep adhering to the styles already used and correct things during review of a pull request or use personal judgement for the minor aesthetics like this?

Reply:
We are never going to have the chance to fix the style in one wholesome swoop, since it would just mess up the history too much. So the rule of thumb is, when you're modifying a line already, touch it up stylistically as well. The rule in the PR template is not meant to be read on a line-by-line basis. You can see from the nearby code that we like spacing around our operators too.

At one point we'll define it and enforce it on PRs, but that's a TODO noted in #161.

Reply:
Thanks for the clarity!